### PR TITLE
Tweak: Usage - Add count per post_status for library [ED-2516]

### PR DESCRIPTION
**Pull request by iNewLegend**
~Depends on https://github.com/elementor/elementor/pull/14545~
~Depends on https://github.com/elementor/elementor/pull/14930~

### CODE REVIEW

File: schemas/usage.json [hunk 1]
```diff
@@ -606,9 +609,8 @@
 					"patternProperties": {
 						"([^\\s]+)": {
 							"$id": "#/usages/library/$template_type",
-							"$comment": "TODO: Should be number",
-							"title": "Count of template-type usage",
-							"type": "string"
+							"$ref": "#/definitions/count_per_post_status",
+							"title": "Libraries per template type"
```
Comment (matipojo) [inline schemas/usage.json:613]: Hi, we don't want to override the existing. instead, add the new data in the following format:

```
	"usages": {
		"library": {
			"kit": "1",
			"page": "7",
			"section": "1"
		},
+		"library-details": {
+			"kit": {
+				"draft": 1,
+				"publish": 2,
+			},
+			"page": {
+				"draft": 2,
+				"publish": 5,
+			},
+			"section":  {
+				"publish": 1,
+			},
+		},
```

**Review by matipojo — CHANGES_REQUESTED** [review]

File: schemas/usage.json (hunk 1 above)
Comment (iNewLegend) [reply schemas/usage.json:633]: Handled.

File: schemas/usage.json [hunk 2]
```diff
@@ -45,7 +45,7 @@
 			"type": "object",
 			"patternProperties": {
 				".*\\.php$": {
-					"$id": "#/system/plugins/active_plugins/$plugin",
+					"$id": "#/definitions/plugins/plugin",
```
Comment (matipojo) [inline schemas/usage.json:48]: Why?

**Review by matipojo — APPROVED** [review]

File: schemas/usage.json (hunk 2 above)
Comment (iNewLegend) [reply schemas/usage.json:48]: It was added with wrong *id* in the past, i noticed it here and fixed.

**Review by matipojo — APPROVED** [review]

**Review by matipojo — CHANGES_REQUESTED** [review]
Please fix conflict

File: tests/phpunit/schemas/usage.json [hunk 3]
```diff
@@ -613,6 +617,24 @@
...
+					"title": "The library details schema",
+					"description": "The usage of library templates by template-type and post-status, the value is the count.",
+					"patternProperties": {
+						"([^\\s]+)": {
+							"$id": "#/usages/library-details/$template_type",
+							"$ref": "#/definitions/count_per_post_status",
+							"title": "Libraries per template type"
+						}
+					},
+					"additionalProperties": false
```
Comment (matipojo) [inline tests/phpunit/schemas/usage.json:636]: Please add an example

Comment (iNewLegend) [reply tests/phpunit/schemas/usage.json:711]: Handled.

**Review by StyleShit — APPROVED** [review]

File: tests/phpunit/schemas/readme.md [hunk 4]
```diff
@@ -1,2 +1,3 @@
 # Changelog
-`3.3.0` - Added `usage/non-elementor-post` for count posts that not built with Elementor.
+* `3.3.0` - Added `usage/non-elementor-post` for count posts that not built with Elementor.
```
Comment (matipojo) [inline tests/phpunit/schemas/readme.md:2]:
```suggestion
* `3.3.0` - Added `usage/non-elementor-posts` for count posts that not built with Elementor.
```

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by matipojo — APPROVED** [review]

**Pull request merged**